csrankings-i.csv

```diff
@@ -407,5 +408,5 @@ Iyad A. Kanj,DePaul University,http://www.cdm.depaul.edu/about/Pages/People/facu
 Iyad Kanj,DePaul University,http://www.cdm.depaul.edu/about/Pages/People/facultyinfo.aspx?fid=272,NOSCHOLARPAGE
 Iyad Katib,King Abdulaziz University,http://iyadkatib.com,DN3hbOsAAAAJ
 Iyad Ouaiss,Lebanese American University,http://services.sea.lau.edu.lb/faculty/iouaiss,Bs47qDUAAAAJ
-Izidor Gertner,CUNY,https://www.ccny.cuny.edu/profiles/izidor-gertner,-3lkqowAAAAJ
+Izidor Gertner,CUNY,https://www.ccny.cuny.edu/profiles/izidor-gertner,3lkqowAAAAJ

```

Comment:
Broken ID, see above.

Reply:
It was a regex mistake done by the author. It is fixed in the current push.